scene/gui/code_edit.h

```diff
@@ -427,7 +435,7 @@ class CodeEdit : public TextEdit {
 
 	void request_code_completion(bool p_force = false);
 
-	void add_code_completion_option(CodeCompletionKind p_type, const String &p_display_text, const String &p_insert_text, const Color &p_text_color = Color(1, 1, 1), const Ref<Resource> &p_icon = Ref<Resource>(), const Variant &p_value = Variant::NIL);
+	void add_code_completion_option(CodeCompletionKind p_type, const String &p_display_text, const String &p_insert_text, const Color &p_text_color = Color(1, 1, 1), const Ref<Resource> &p_icon = Ref<Resource>(), const Variant &p_value = Variant::NIL, int p_location = LOCATION_OTHER);
```

Comment:
Why not use `CodeCompletionLocation` as type? Hopefully this might also properly expose the type in the docs, as currently there's no link made between that parameter and the enum.

Reply:
unfortunately this is not a `BitField` either